[2024-11-01T05:09:28.456+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-01T05:09:28.473+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: self-tutorial-dag.creating_table manual__2024-11-01T05:09:26.193368+00:00 [queued]>
[2024-11-01T05:09:28.480+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: self-tutorial-dag.creating_table manual__2024-11-01T05:09:26.193368+00:00 [queued]>
[2024-11-01T05:09:28.481+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-01T05:09:28.496+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): creating_table> on 2024-11-01 05:09:26.193368+00:00
[2024-11-01T05:09:28.502+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1784) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-01T05:09:28.504+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'self-tutorial-dag', 'creating_table', 'manual__2024-11-01T05:09:26.193368+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/self-tutorial-dag.py', '--cfg-path', '/tmp/tmpkglidsq1']
[2024-11-01T05:09:28.504+0000] {standard_task_runner.py:105} INFO - Job 42: Subtask creating_table
[2024-11-01T05:09:28.504+0000] {standard_task_runner.py:72} INFO - Started process 1785 to run task
[2024-11-01T05:09:28.540+0000] {task_command.py:467} INFO - Running <TaskInstance: self-tutorial-dag.creating_table manual__2024-11-01T05:09:26.193368+00:00 [running]> on host 2839f63d218a
[2024-11-01T05:09:28.621+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='self-tutorial-dag' AIRFLOW_CTX_TASK_ID='creating_table' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T05:09:26.193368+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-01T05:09:26.193368+00:00'
[2024-11-01T05:09:28.623+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-01T05:09:28.641+0000] {sql.py:266} INFO - Executing: 
            create table if not exists naver_search_result(
                title TEXT,
                address TEXT,
                category TEXT,
                description TEXT,
                link TEXT
            )
        
[2024-11-01T05:09:28.653+0000] {base.py:84} INFO - Retrieving connection 'db_postgres'
[2024-11-01T05:09:28.667+0000] {base.py:84} INFO - Retrieving connection 'db_postgres'
[2024-11-01T05:09:28.671+0000] {sql.py:509} INFO - Running statement: 
            create table if not exists naver_search_result(
                title TEXT,
                address TEXT,
                category TEXT,
                description TEXT,
                link TEXT
            )
        , parameters: None
[2024-11-01T05:09:28.681+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-01T05:09:28.681+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=self-tutorial-dag, task_id=creating_table, run_id=manual__2024-11-01T05:09:26.193368+00:00, execution_date=20241101T050926, start_date=20241101T050928, end_date=20241101T050928
[2024-11-01T05:09:28.723+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-01T05:09:28.754+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-01T05:09:28.756+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
